worlds/lingo/player_logic.py

```diff
@@ -36,6 +37,27 @@ class PlayerLocation(NamedTuple):
     access: AccessRequirements
 
 
+class ProgressiveItemBehavior(Enum):
+    DISABLE = 1
+    SPLIT = 2
+    PROGRESSIVE = 3
+
+
+def should_split_progression(progression_name: str, world: "LingoWorld") -> ProgressiveItemBehavior:
+    if progression_name == "Progressive Orange Tower":
+        if world.options.progressive_orange_tower:
+            return ProgressiveItemBehavior.PROGRESSIVE
+        else:
+            return ProgressiveItemBehavior.SPLIT
+    elif progression_name == "Progressive Colorful":
+        if world.options.progressive_colorful:
+            return ProgressiveItemBehavior.PROGRESSIVE
+        else:
+            return ProgressiveItemBehavior.SPLIT
+
+    return ProgressiveItemBehavior.PROGRESSIVE
```

Comment:
Is this supposed to be `return ProgressiveItemBehavior.DISABLE` instead? I don't see `DISABLE` being used anywhere, and I'd assume it is the default case

Reply:
PROGRESSIVE is the default case because most progressive items are always progressive. DISABLE fully removes the progressive item from the pool. It is not used at all yet, but it is used in the sunwarp shuffling PR, because it is possible to make sunwarp access individual, progressive, or completely disabled.